examples/llamaindex_local_demo.py

```
@@ -0,0 +1,64 @@
+"""
+Quick Start Demo: LlamaIndex RAG without OpenAI API
+
+This demo shows how to use LlamaIndex with local embeddings
+when OpenAI API is unavailable.
+"""
+
+import asyncio
+from pathlib import Path
```

Flake8 (7.3.0)

[error] 'pathlib.Path' imported but unused

(F401)